components/modules/NewReleases/index.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useState } from "react";
+import { songs } from "../utils/dummy";
+import { VDots, VPause, VPlay, VStar, VWave } from "../__modules__/_Vectors";
+import liveBg from "../../../public/livebg.jpg";
+import AudioPlayer from "../utils/config/audioPlayer";
+import equalizer from "../static/images/equalizer.gif";
+
+interface TrackItems{
+  [index: number]:{title: string;
+  artist: string;
+  audioSrc: string;
+  image: string;}
+}
+
+
+const NewReleases = () => {
+  let songsData:TrackItems=songs;  
```

Comment:
As this line is a constant, we don't need it to be reloaded every time the component reload. You can write it out side of the component